ftw/testbrowser/core.py

```diff
@@ -1018,6 +1018,22 @@ def _normalize_url(self, url_or_object, view=None):
 
         return url
 
+    @staticmethod
+    def _correct_webdav_xml(html):
+        """
+        This method fixes erroneous namespacing in WebDav documents
+        generated by Plone ZServer.
+        """
+        if isinstance(html, StringIO):
+            html.seek(0)
+            html = html.getvalue()
+
+        if isinstance(html, (unicode, str)) and 'xmlns:d="DAV:"' in html:
```

Comment:
you could use `basestring` here, the common superclass of those two. see https://docs.python.org/2/library/functions.html#basestring.